.github/workflows/ci.yml

```diff
@@ -77,11 +77,5 @@ jobs:
       test_environment: ${{ matrix.test_environment }}
     secrets: inherit
 
-  spell-check:
-    uses: ./.github/workflows/spellcheck.yml
-
```

Comment:
In spite of how happy this makes me, don't we still need the spellcheck ?

Reply:
You're right. This was from when it seemed like maybe ruff handled that too. Forgot to revert the changes. Thanks for catching!